widget/tree.go

```diff
@@ -694,7 +695,9 @@ func (r *treeNodeRenderer) Objects() (objects []fyne.CanvasObject) {
 
 func (r *treeNodeRenderer) Refresh() {
 	if c := r.treeNode.content; c != nil {
-		c.Refresh()
+		if f := r.treeNode.tree.UpdateNode; f != nil {
+			f(r.treeNode.uid, r.treeNode.isLeaf, c)
```

Comment:
The second parameter of `UpdateNode` is `isBranch`

Reply:
good catch thanks, will fix the logic